src/modules/ce/components/ProjectReferralsTable.tsx

```diff
@@ -16,36 +16,58 @@ import {
 } from '@/types/gqlTypes';
 import { generateSafePath } from '@/utils/pathEncoding';
 
-const COLUMNS: ColumnDef<CeReferralTableFieldsFragment>[] = [
-  {
+export const REFERRAL_COLUMNS: Record<
+  string,
+  ColumnDef<CeReferralTableFieldsFragment>
+> = {
+  client: {
     header: 'Client',
-    render: (referral) => clientNameFromRecordWithOptionalClient(referral),
+    render: (referral: CeReferralTableFieldsFragment) =>
+      clientNameFromRecordWithOptionalClient(referral),
     key: 'name',
     sticky: 'left',
   },
-  {
+  opportunity: {
     header: 'Opportunity',
     key: 'opportunity',
-    render: (referral) => referral.opportunity.name,
+    render: (referral: CeReferralTableFieldsFragment) =>
+      referral.opportunity.name,
   },
-  {
-    header: 'Started',
-    key: 'started',
-    render: (referral) => (
+  date: {
+    header: 'Referral Date',
+    key: 'date',
+    render: (referral: CeReferralTableFieldsFragment) => (
       <RelativeDateDisplay dateString={referral.createdAt} />
```

Comment:
"Referral Date" doesn't feel like the right header for a relative date description string. Another thing to align on with design. I would suggest just using the date-only format (`parseAndFormatDate`) to be consistent with the other tables on the Client dashboard. (Services, Assessments, Case notes -- they all have a formatted date as the first column.)